### ISSUE TITLE
Issues regarding lack of overshoot, making rounded characters look smaller

### ISSUE THREAD

**Issue by forsgren** [jpt/barlow#74]
I find that the characters (mainly caps) that are round at the top or base does not have the visual "overflow" needed for them to visually feel like they are the same height as other letters. This causes S, O, 0 (zero), 8, 9 and so on to look smaller. See attached screenshot. Using capital "i" as a "boxy" comparison.

<img width="334" alt="Screen Shot 2021-11-25 at 17 50 28" src="https://user-images.githubusercontent.com/164518/143479574-ac6fc9dc-ea59-427c-ab98-d3beb924bfbe.png">

The string in this case: `IIOII0ISI819II`

The "round characters" need some [overshoot love](https://en.wikipedia.org/wiki/Overshoot_(typography)).

Edit: This seems to be an issue also in 1.5 (where the screenshot is from)

**Comment by jpt** [comment]
Hi @forsgren thanks for the feedback. I agree with you. IIRC (I’m not at a computer where I can inspect right now) barlow’s overshoots are pretty small (8upm or so). I wonder if this is as much of a visual issue in the upright letters or if the faux-italic is causing downstrokes to appear a bit heavier and thus more in need of overshoot? 

if it’s an issue that requires different overshoots for uprights I may have to address this via #11 

I’ll take a closer look later in the week and give you an update based on what I find; thanks again!

**Comment by forsgren** [comment]
To me it seems as if the upright letters have the same issue. Maybe a tad bit less noticeable, but I still find them to appear a bit smaller. 

I really appreciate your work with this typeface!

**Comment by jpt** [comment]
No, it's noticeable, you are correct about the uprights. I just had a look. At first glance, I would say the condensed black rounds need to go from could go from 8UPM to at least 10UPM, and the regular width could potentially go all the way up to 14UPM or so. I will play around with them. This is great feedback, thanks again for opening the issue.

**Comment by jpt** [comment]
hi @forsgren — can you take a look at these binaries and let me know if the situation has improved? it's looking about right in my environment
[Barlow1.5-RC1.zip](https://github.com/jpt/barlow/files/7655026/Barlow1.5-RC1.zip)
.

**Comment by forsgren** [comment]
@jpt It seems to have improved :) To me it still looks like the rounded characters still might need _slightly_ more overshoot. Not sure if you agree?

<img width="358" alt="Screen Shot 2021-12-09 at 09 24 14" src="https://user-images.githubusercontent.com/164518/145360830-991f5782-fb75-4a0f-8240-131f1db82216.png">

Same test string as previously `IIOII0ISI819II`

**Comment by jpt** [comment]
Thanks @forsgren; I'm in agreement after looking away from it for a little bit. I will probably go from 12UPM to no more than 15, playing around with it now, will give you an update when there's a new set of files to look at.
